test/src/test/java/hudson/cli/RunRangeCommand2Test.java

```diff
@@ -98,7 +98,7 @@ public class RunRangeCommand2Test {
 
     @Test public void dummyRangeShouldSuccessEvenTheBuildIsStuckInTheQueue() throws Exception {
         FreeStyleProject project = j.createFreeStyleProject("aProject");
-        project.getBuildersList().add(new Shell("echo 1\nsleep 10s"));
+        project.getBuildersList().add(new Shell("echo 1\nsleep 10"));
```

Comment:
This looks like it would fail on Windows?

Reply:
All tests are passing on ci.j